### PR TITLE
Module 4: advanced components

### PR DESCRIPTION
**Pull request by bioN1ck**

### CODE REVIEW

File: src/components/MovieDetails/MovieDetails.tsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
 import './MovieDetails.scss';
 import { Movie } from '../../models/movie.model';
 
+export const handleDuration = (duration: number) => {
```
Comment (melk0sha) [inline src/components/MovieDetails/MovieDetails.tsx]: `handleSomething` usually used to handle React events (onClick, onSubmit, custom onDownload), but as for utils it's better to use common names like `getDuration`

File: src/App.tsx [hunk 2]
```diff
@@ -24,6 +29,16 @@ function App() {
         <Counter/>
         <Search initialValue={''} onSearch={handleSearch}/>
         <GenreSelect genres={genres} onSelect={handleGenreSelect}/>
+        <Dialog
+          trigger={<Button>Open a dialog</Button>}
+          title={'Add movie'}
+          onClose={() => {}}
```
Comment (melk0sha) [inline src/App.tsx]: It would be nice to have a handler implementation here to close the dialog

**Review by melk0sha — APPROVED** [review]

File: src/components/MovieDetails/MovieDetails.tsx (hunk 1 above)
Comment (bioN1ck) [reply src/components/MovieDetails/MovieDetails.tsx]: Fixed

File: src/App.tsx (hunk 2 above)
Comment (bioN1ck) [reply src/App.tsx]: Deleted

**Pull request merged**